pkg/apiserver/apiserver.go

```diff
@@ -126,6 +128,10 @@ type CompletedConfig struct {
 // Complete fills in any fields not set that are required to have valid data. It's mutating the receiver.
 func (c *Config) Complete() CompletedConfig {
 	c.GenericConfig.EffectiveVersion = version.DefaultBuildEffectiveVersion()
+	c.GenericConfig.SkipOpenAPIInstallation = true
+	// required for InstallAPIGroup
+	c.GenericConfig.OpenAPIV3Config = genericapiserver.DefaultOpenAPIV3Config(generated.GetOpenAPIDefinitions, openapi.NewDefinitionNamer(Scheme))
+
```

Comment:
and skipping OpenAPI installation

Reply:
Can you explain the motivation behind this? What problem is this solving?

Reply:
We need the OpenAPIV3Config so we do not fail here:

https://github.com/openshift/generic-admission-server/blob/master/vendor/k8s.io/apiserver/pkg/server/genericapiserver.go#L813-L815

Unfortunately I forgot, how I was testing it, but I cannot reproduce the error anymore. Mainly because it was also added here by Arda:

https://github.com/openshift/generic-admission-server/blob/8dcc3c9b298fefaf4d31c60001907222bf2c3f83/pkg/cmd/server/start.go#L131

Either I tested the options separately, or I had an older master without Arda's commit.

I added SkipOpenAPIInstallation, because IIRC the openapi aggregation is done only for CRDs, and extension apiservers, and not webhooks.